iq/categories/tests.py

```diff
@@ -1,3 +1,137 @@
+import datetime
+from unittest import TestCase, mock
+from unittest.mock import patch
 from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from django.contrib.auth.models import User
+from model_mommy.mommy import make, prepare
+from iq.questions.models import Question, CategoryQuestion
+from iq.categories.models import Category
+from iq.categories.forms import CategoryForm
+from iq.utils.tests import IqCustomTest
 
-# Create your tests here.
+
+class CategoryModelsTest(IqCustomTest):
+    """
+    Tests category models for appropriate data
+    """
+    def setUp(self):
+        super(CategoryModelsTest, self).setUp()
+
+    def test_category_model(self):
```

Comment:
Your Category model is simple. The only thing you need to test (right now, at least) is __str__. Testing the constructor and the save method isn't necessary, since you didn't override it.